ansible/group_vars/dash.nimbus.yml

```diff
@@ -30,9 +30,9 @@ oauth_upstream_addr: '{{ kibana_cont_name }}'
 oauth_upstream_port: '{{ kibana_cont_port }}'
 oauth_local_port: 4180
 oauth_provider: 'keycloak-oidc'
-oauth_id: '{{ lookup("bitwarden", "nimbus/kibana/oauth", field="client-id") }}'
-oauth_secret: '{{ lookup("bitwarden", "nimbus/kibana/oauth", field="secret") }}'
-oauth_cookie_secret: '{{ lookup("bitwarden", "nimbus/kibana/oauth", field="cookie-secret") }}'
+oauth_id:            '{{lookup("vault", "kibana", field="client-id",     stage="dash", env="nimbus")}}'
```

Comment:
`env` and `stage` are available to the lookup plugin. you need to specify them only if you override them

Reply:
Also the secret could be called `kibana/oauth` since these are just `oauth` credentials and not `kibana` credentials